microsetta_admin/templates/bulk_scan.html

```diff
@@ -0,0 +1,229 @@
+            val += "<br>Source: " + event.target.children[3].innerHTML;
+            document.getElementById("codeDetails").innerHTML = val;
+            document.getElementById("summaryView").style.display = "block";
+            document.getElementById('recorded_type').value = event.target.children[3].innerHTML;
+            document.getElementById('sample_status').value = event.target.children[2].innerHTML;
+            document.getElementById("tubeCode").href = "/scan?sample_barcode=" + event.target.children[0].innerHTML;
+        }
+
+    }
+
+    function hideSummary()
+    {
+        if(!clicked)
+        {
+            document.getElementById("summaryView").style.display = "none";
+        }
+    }
+
+    function toggleSummary()
+    {
+        if(clicked)
+        {
+            clicked = false;
+            hideSummary();
+        }
+        else
+        {
+            clicked = true;
+        }
+    }
+
+    function showNotesField()
+    {
+        document.getElementById("summaryView").style.height = "40vh";
+        document.getElementById('btnNotes').style.display = "none";
+        document.getElementById("notesBox").style.display = "block";
+        document.getElementById('btnAddNotes').style.display = "block";
+    }
+
+    function hideNotesField()
+    {
+        document.getElementById("summaryView").style.height = "27vh";
+        document.getElementById('btnNotes').style.display = "block";
+        document.getElementById("btnNotes").value = "Notes Saved Edit/View?";
+        document.getElementById("notesBox").style.display = "none";
+        document.getElementById('btnAddNotes').style.display = "none";
+    }
+
+</script>
+<style>
+    .legends{
+        border: 2px solid black;
+        border-collapse: collapse;
+    }
+
+    .data_table{
+        width: 50%;
+        border: 2px solid black;
+        border-collapse: collapse;
+    }
+
+    .data_table > tbody > tr > td, .data_table > tbody > tr > th{
+        border: 1px solid black;
+        text-align: center;
+        width: 100px;
+    }
+
+    .data_table > tbody > tr > td{
+        color: white;
+    }
+
+    #summaryView{
+        display: none;
+        width: 50%;
+        height: 25vh;
+        border: 2px solid black;
+        padding: 2% 2% 2% 2%;
+        margin-top: 2%;
+        margin-left: 2%;
+    }
+
+    #tubeCode{
+        text-decoration: underline;
+    }
+
+    #notesBox{
+        display: none;
+    }
+
+    #btnAddNotes{
+        display: none;
+    }
+
+</style>
+{% endblock %}
+{% block content %}
+<h3>Microsetta Scan - Bulk</h3>
+
+{% if stage == "Scanning" %}
+<div>
+    <form method="POST" action="/scan-bulk" enctype="multipart/form-data">
+        <table>
+            <tr>
+                <td><label for="txtfilePath">Import .csv:     </label></td>
+                <td><input type="file" name="file_picker" id="file_picker" onchange="update_file()" accept=".csv" required></td>
```

Comment:
Is the update_file() function still necessary after changing this to an input type="file"?

Reply:
Removed unnecessary code